src/x11/xf86Wacom.c

```diff
@@ -397,11 +397,9 @@ valuatorNumber(enum WacomAxisType which)
 }
```

Comment:
While resolving conflicts, you forgot to add the scroll axes to the switch in `valuatorNumber()`. See 07581618a0d26a8135cde4fd930c3eb432d62461

Reply:
squashed in, thanks!